library/src/main/java/com/okta/appauth/android/OktaAppAuth.java

```diff
@@ -565,6 +587,30 @@ private void doAuth(PendingIntent completionIntent, PendingIntent cancelIntent)
                 mAuthIntent.get());
     }
 
+    private void doEndSession(PendingIntent completionIntent, PendingIntent cancelIntent) {
+        Log.d(TAG, "Starting end session flow");
+        try {
+            mAuthIntentLatch.await();
+        } catch (InterruptedException ex) {
+            Log.w(TAG, "Interrupted while waiting for auth intent");
+        }
+
+        EndSessionRequest request = new EndSessionRequest(
+                mAuthStateManager.getCurrent().getAuthorizationServiceConfiguration(),
+                mAuthStateManager.getCurrent().getIdToken(),
+                mConfiguration.getmEndSessionRedirectUri());
+
+        CustomTabsIntent endSessionInent;
```

Comment:
move declaration to line 608 if possible to make it more clear

Reply:
Done